test/infrastructure/vcsim/controllers/vmip_controller.go

```diff
@@ -79,7 +79,7 @@ func (r *vmIPReconciler) ReconcileIP(ctx context.Context) (ctrl.Result, error) {
 	if err != nil {
 		return reconcile.Result{}, errors.Wrapf(err, "failed to PowerOff vm")
 	}
-	if err = task.WaitEx(ctx); err != nil {
+	if err = task.Wait(ctx); err != nil { // deprecation on Wait is going to be removed, see https://github.com/vmware/govmomi/issues/3394
```

Comment:
```suggestion
	if err = task.Wait(ctx); err != nil { //nolint:staticcheck // deprecation on Wait is going to be removed, see https://github.com/vmware/govmomi/issues/3394
```

Also for the others :-)

Reply:
uhmm, it seems that make lint locally disagree with make lint in CI

Reply:
@fabriziopandini I'm getting this output locally (same as in CI)

```yaml
INFO [runner] linters took 9.066724209s with stages: goanalysis_metalinter: 9.058008375s
test/infrastructure/vcsim/controllers/vmip_controller.go:82:11: SA1019: task.Wait is deprecated: Please use WaitEx instead. (staticcheck)
        if err = task.Wait(ctx); err != nil { // deprecation on Wait is going to be removed, see https://github.com/vmware/govmomi/issues/3394
                 ^
test/infrastructure/vcsim/controllers/vmip_controller.go:120:11: SA1019: task.Wait is deprecated: Please use WaitEx instead. (staticcheck)
        if err = task.Wait(ctx); err != nil { // deprecation on Wait is going to be removed, see https://github.com/vmware/govmomi/issues/3394
                 ^
test/infrastructure/vcsim/controllers/vmip_controller.go:129:11: SA1019: task.Wait is deprecated: Please use WaitEx instead. (staticcheck)
        if err = task.Wait(ctx); err != nil { // deprecation on Wait is going to be removed, see https://github.com/vmware/govmomi/issues/3394
                 ^
INFO File cache stats: 70 entries of total size 561.6KiB
```

Maybe clean bin or something helps?

Reply:
golangci-lint cache clean did the trick